### ISSUE TITLE
allow user to specify ancillary files path

### ISSUE THREAD

**Issue by paolap** [ACCESS-Community-Hub/ACCESS-MOPPeR#90]
- add anvils_path to config.yaml 
-  update setup_env() function
-  update calculations.py where ancillary files are used
So far we're indicating path but we might have to change this to allow single files

**Comment by paolap** [comment]
Added ancillary path to conf file. This is probably not sufficient and we should also indicate ancillary file names as we cannot know before end what ancils any experiment will use.
However, it might be possible that in the model setup they're all called in the same way, for example ice anvil file is called grid.nc

**Comment by paolap** [comment]
Added ancil_ocean and ancil_ice to conf.yaml file.
These are used in get_coords() when they are needed to retrieve boundaries of grid lat/lon. 
I used the ancil_<realm>  pattern, so we can use cox.obj['realm'] to pick the right file. 
I also extended the dictionary to map the name of latitude/longitude in the input file to the vertices/bnds in the ancil file. These are usually different in a way that is not possible to work them out, they need to be explicitly defined. 
It might be better to export them as a yaml file so they are more visible.
